packages/frontend/pages/items/create-auction.tsx

```diff
@@ -2,19 +2,44 @@ import { wrapper } from 'store';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import { Layout } from '@components/layout';
 import { SavePost } from '@components/save-post';
-import { ProductType } from '@vse-bude/shared';
+import type { UserResponseDto } from '@vse-bude/shared';
+import { Http, ProductType } from '@vse-bude/shared';
+import { withPublic } from '@hocs';
+import { getCurrentUserSSR } from 'store/auth';
+import { AuthHelper, CookieStorage } from '@helpers';
+import { Routes } from '@enums';
 
-export const getServerSideProps = wrapper.getServerSideProps(
-  (_store) =>
-    async ({ locale }) => ({
+export const getServerSideProps = withPublic(
+  wrapper.getServerSideProps((store) => async (ctx) => {
+    const { locale } = ctx;
+
+    const cookieStorage = new CookieStorage(ctx);
+    const auth = new AuthHelper(cookieStorage);
+    const http = new Http(process.env.NEXT_PUBLIC_API_ROUTE, locale, auth);
+
+    const { payload } = (await store.dispatch(getCurrentUserSSR(http))) as {
+      payload: UserResponseDto;
+    };
+
+    if (!payload.phoneVerified || !payload.emailVerified) {
+      return {
+        redirect: {
+          destination: Routes.NOT_FOUND,
+        },
+        props: {},
+      };
+    }
```

Comment:
I think we need to allow users to open "make a product page". The user should be authorised. If they authorised but not verified, than we will show warning: "Hi, your phone number is not verified, click here to verify"